doc/ref_model/chapters/chapter07.md

```diff
@@ -96,6 +98,41 @@ At a high level, the following areas/requirements cover workload security for a
 * Workload owner owns workload design change process
 * Workload owner owns workload software update process
   
+ <a name="7.3.3"></a> 
+ ### 7.3.3 Common Security Standards
+
+The Cloud Infrastructure Reference Model and the supporting architectures are not only required to optimally support networking functions, but they must be designed with common security principles and standards from inception.  These best practices must be applied at all layers of the infrastructure stack and across all points of interconnections with outside networks, APIs and contact points with the NFV network functions overlaying or interacting with that infrastructure.
+Standards organizations with recommendations and best practices, and certifications that need to be taken into consideration include the following examples. However this is by no means an exhaustive list, just some of the more important standards in current use.
+
+* Center for Internet Security - https://www.cisecurity.org/
+* Cloud Security Alliance - https://cloudsecurityalliance.org/
+* Open Web Application Security Project https://www.owasp.org
+* The National Institute of Standards and Technology (NIST) (US Only)
```

Comment:
"US only" implies that others can't use. If we delete "only" would that work?

Reply:
@pgoyal01 Yes, the standard is widely used
(US only) now removed